tests/unit/lib/init/test_cli_template_modifier.py

```diff
@@ -328,3 +351,183 @@ def test_must_pass_sanity_check(self, parse_yaml_file_mock):
         parse_yaml_file_mock.return_value = {"add: add_value"}
         result = template_modifier._sanity_check()
         self.assertTrue(result)
+
+    @patch(
+        "samcli.lib.init.template_modifiers.application_insights_template_modifier.ApplicationInsightsTemplateModifier._get_template"
+    )
+    def test_must_add_application_insights_monitoring(self, get_template_patch):
```

Comment:
This class (or file) already has too many lines. Since these test will cover a new implementation, can we move them into their own file?

Reply:
Sure, will change that

Reply:
@vsukameti same as previous, were you able to push your changes to address this comment?

Reply:
Yes, moved tests to a new file, "test_application_insights_template_modifier"